packages/gatsby-plugin-feed/src/plugin-options.js

```diff
@@ -0,0 +1,17 @@
+import Joi from "joi"
```

Comment:
I never knew Joi was a thing. Cool!

Reply:
It's amazing :) We use it in gatsby core, and it's really nice to warn/error when we don't get the data we expect!